packages/core/src/indexing/sqlite/__tests__/insertion-order.test.ts

```diff
@@ -0,0 +1,111 @@
+import { DataSource } from 'typeorm'
+import tmp from 'tmp-promise'
+import knex from 'knex'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { SqliteIndexApi } from '../sqlite-index-api.js'
+import { readFixture } from './read-fixture.util.js'
+import { chunks } from './chunks.util.js'
+import { InsertionOrder } from '../insertion-order.js'
+
+const MODEL_ID = 'kjzl6cwe1jw145m7jxh4jpa6iw1ps3jcjordpo81e0w04krcpz8knxvg5ygiabd'
+const MODELS_TO_INDEX = [StreamID.fromString(MODEL_ID)]
+const MODEL = MODELS_TO_INDEX[0]
+
+let EXPECTED: Array<string>
+
+let tmpFolder: tmp.DirectoryResult
+let dataSource: DataSource
+let order: InsertionOrder
+
+beforeEach(async () => {
+  tmpFolder = await tmp.dir({ unsafeCleanup: true })
+  dataSource = new DataSource({
+    type: 'sqlite',
```

Comment:
You could just replace this with:

`dataSource = knex({
  client: 'sqlite3', // or 'better-sqlite3'
  connection: {
    filename: "${tmpFolder.path}/tmp-ceramic.sqlite"
  }
});`

and remove typeorm then.